cmd/yace/main.go

```diff
@@ -191,7 +133,7 @@ func main() {
 	http.HandleFunc("/metrics", func(w http.ResponseWriter, r *http.Request) {
 		if !(*decoupledScraping) {
 			newRegistry := prometheus.NewRegistry()
-			updateMetrics(newRegistry, now)
+			exporter.UpdateMetrics(config, newRegistry, now, *metricsPerQuery, *fips, *debug, *floatingTimeWindow, *labelsSnakeCase)
```

Comment:
A lot of parameters are passed that could be part of an operational configuration struct.